Neos.ContentRepository/Classes/Domain/Context/DimensionSpace/ContentSubgraph.php

```diff
@@ -0,0 +1,102 @@
+<?php
+namespace Neos\ContentRepository\Domain\Context\DimensionSpace;
+
+/*
+ * This file is part of the Neos.ContentRepository package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\ContentRepository\Domain\Context\Dimension;
+use Neos\ContentRepository\Domain;
+
+/**
+ * The content subgraph domain model
```

Comment:
One of the main APIs to interact with the new CR on the write side.

DO NOT CONFUSE with ContentSubgraph on READ side!

Reply:
TODO maybe new name?